src/garage/envs/point_env.py

```diff
@@ -27,10 +27,12 @@ def __init__(
             goal=np.array((1., 1.), dtype=np.float32),
             done_bonus=0.,
             never_done=False,
+            action_scale=1.,
```

Comment:
please don't scale actions inside environments. it's up to the algorithm to do this.